### PR TITLE
Add latency to the !ping response

### PR DESCRIPTION
**Pull request by Scott-Caldwell**

### CODE REVIEW

File: Modix.Bot/Modules/PingModule.cs [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+        {
+            DiscordClient = discordClient;
+        }
+
         [Command("ping")]
         public Task Ping() =>
-            ReplyAsync("Pong!");
+            ReplyAsync($"Pong! ({DiscordClient.Latency} ms)");
+
+        protected DiscordSocketClient DiscordClient { get; }
```
Comment (calledude) [inline Modix.Bot/Modules/PingModule.cs]: Huh? Why `protected`?

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/PingModule.cs]: Why not?

Comment (calledude) [reply Modix.Bot/Modules/PingModule.cs]: What are you expecting to inherit from `PingModule`?

Comment (calledude) [reply Modix.Bot/Modules/PingModule.cs]: There's no reason to expose this field

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/PingModule.cs]: In that case, would you recommend sealing the class and making the property/field private?

Comment (calledude) [reply Modix.Bot/Modules/PingModule.cs]: Yes, I realize I'm being super nitpicky here, but there's not a lot to pick on when the PR is 10 lines 😬

Comment (Scott-Caldwell) [reply Modix.Bot/Modules/PingModule.cs]: Updated

**Pull request merged**